src/ServiceControl.Audit.Persistence.RavenDB/CustomChecks/CheckFreeDiskSpace.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace ServiceControl.Audit.Persistence.RavenDB.CustomChecks
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus.CustomChecks;
+    using NServiceBus.Logging;
+    using RavenDB;
+
+    class CheckFreeDiskSpace(DatabaseConfiguration databaseConfiguration) : CustomCheck("ServiceControl.Audit database",
+        "Storage space", TimeSpan.FromMinutes(5))
+    {
+        public override Task<CheckResult> PerformCheck(CancellationToken cancellationToken = default)
+        {
+            if (Logger.IsDebugEnabled)
+            {
+                Logger.Debug($"Check ServiceControl data drive space remaining custom check starting. Threshold {percentageThreshold:P0}");
+            }
+
+            if (!databaseConfiguration.ServerConfiguration.UseEmbeddedServer)
+            {
+                return CheckResult.Pass;
+            }
+
+            if (dataPathRoot == null)
+            {
+                throw new Exception($"Unable to find the root of the data path {dataPathRoot}");
```

Comment:
Seems like we need to keep the data path value instead of just storing `dataPathRoot` because this error message is nonsensical now!

Looks like this applies to all 4 checks.